andela/travel_tool_front
# #161875838 User can logout end to end test

#### What does this PR do?
- Write test for logout functionality 

#### Description of Task to be completed?
- Write test for logout functionality

#### How should this be manually tested?
- Open the terminal  
- Run `yarn end2end` to run the test with browser mode
- When redirected to the browser, select `component/Requests/logout.spec.js`
- The tests should run and pass
- To run test only in the console, run `yarn end2end:headless` 

#### Any background context you want to provide?
- Ensure both the frontend and backend servers are running

#### What are the relevant pivotal tracker stories?
- [161875838](https://www.pivotaltracker.com/story/show/161875838)

#### Screenshots (if appropriate)
- None

#### Questions:
- None
